src/cogs/help.py

```diff
@@ -9,107 +9,48 @@
-            cmd_info = {
-                "name": cmd.name,
-                "usage": cmd.usage,
-                "doc_string": cmd.help,
-                "aliases": cmd.aliases,
-            }
-            cmd_item = {cmd.name: cmd_info}
-            cmd_dict[cmd.name] = cmd_info
-            all_cmds[cmd.name] = cmd_info
-
-            cmd_dict.update(cmd_item)
-            all_cmds.update(cmd_item)
-            if not cmd.enabled:
-                disabled_cmds.append(cmd.name)
-                current_cog_disabled_cmds.append(cmd.name)
-            else:
-                enabled_cmds.append(cmd.name)
-
-        cmd_count = len(bot.get_cog(cog).get_commands()) - len(
-            current_cog_disabled_cmds
-        )
-
-        cog_dict[cog] = {
-            "command_count": str(cmd_count),
-            "commands": cmd_dict,
-            "disabled_commands": current_cog_disabled_cmds,
-        }
-
-        cog_dict["disabled_commands"] = disabled_cmds
-        cog_dict["enabled_commands"] = enabled_cmds
-
-    for cmd in bot.walk_commands():
-        if not cmd.name in cmd_dict.keys():
-            misc_cmd_dict[cmd.name] = {
-                "name": cmd.name,
-                "usage": cmd.usage,
-                "doc_string": cmd.help,
-                "aliases": cmd.aliases,
-            }
-
-    misc_dicts = {"misc_commands": misc_cmd_dict, "all_commands": all_cmds}
-    cog_dict.update(misc_dicts)
-
-    return cog_dict
-
-
-def misc_command_str(cog_dict):
+def misc_command_str(cmds):
     """Builds the uncategorized command string for the discord embed description."""
     uncategorized_commands = ""
-    for cmd in cog_dict["misc_commands"]:
+    for cmd in cmds:
         uncategorized_commands += f"`{cmd}`, \n"
     return uncategorized_commands[:-3]
 
 
-def description_string_builder(prefix, parsable_json, disabled_commands):
+def description_string_builder(prefix, cog):
     """Builds the description command string for the discord embed description."""
 
     description_string = ""
-    for item in parsable_json["commands"]:  # builds the embed description
-        doc_item = parsable_json["commands"][item]
-        if doc_item["name"] in disabled_commands:
+    for cmd in cog.walk_commands():  # builds the embed description
+        if not cmd.enabled:
             continue
+        doc_item = cmd.description
         alias_string = ""
-        for alias in doc_item["aliases"]:
+        for alias in cmd.aliases:
             alias_string += f"`{alias}`, " if alias else None
         alias_string = alias_string[:-2]
         dash = ", " if alias_string else ""
-        usage = doc_item["usage"] if doc_item["usage"] else None
+        usage = cmd.usage if cmd.usage else ""
         description_string += (
-            f"`{item}`{dash}{alias_string}:\n"
-            f'{doc_item["doc_string"]}\n'
-            f"```{prefix}{item} {usage}```\n\n"
+            f"`{cmd}`{dash}{alias_string}:\n"
```

Comment:
Wouldn't `cmd` in this case return an object? `cmd.name` would fit better, wouldn't it?